src/main/java/com/js/tictactoe/judge/Judge.java

```diff
@@ -55,16 +55,35 @@ private boolean isWinnerInColumn(int column) {
     }
 
     private boolean isWinnerInDiagonal1(int x, int y) {
```

Comment:
to nie zadziala :P 
np dla:
0 0: X
1 1: X
2 2: X
3 3: O
4 4: X
jezeli do wygrania wymaga min 3